core/src/main/java/org/everit/json/schema/loader/SchemaLoader.java

```diff
@@ -178,6 +180,11 @@ SchemaLoaderBuilder pointerToCurrentObj(List<String> pointerToCurrentObj) {
             return this;
         }
 
+        SchemaLoaderBuilder useDefaults(boolean useDefaults) {
```

Comment:
Would you please add some javadoc? Also, this seems to be the toggle point of the feature, so it would be necessary to make the method `public`, otherwise users won't be able to use this feature. I know there are some builder methods here which are package-private, but unlike this one, those are only for internal use by the library.